.github/workflows/eslint-stylelint.yml

```diff
@@ -0,0 +1,20 @@
+name: Run Eslint and Stylelint
+on: pull_request
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: install, bootstrap
+        run: |
+          npm install
+          npm run bootstrap
+          npm run build
+      - name: Run ESLint
+        run: npm run test:eslint
+      - name: Run stylelint
+        run: npm run test:stylelint
```

Comment:
we should also run `npm run test:jest`

Reply:
Current circleci run this command - 
```
"test": "f () { bash ./scripts/circleci-config.test.sh && npm run test:stylelint  && npm run test:eslint && npm run test:jest ;}; f",
```

Reply:
Alright then 👍 @ali-hosseini-fs

Reply:
@ali-hosseini-fs Can we merge these workflows so that we don't have run `bootstrap` and `build` in every workflow?